src/gui/optionsdialog.cpp

```diff
@@ -704,6 +704,11 @@ void OptionsDialog::loadDownloadsTabOptions()
     connect(m_ui->groupMailNotifAuth, &QGroupBox::toggled, this, &ThisType::enableApplyButton);
     connect(m_ui->mailNotifUsername, &QLineEdit::textChanged, this, &ThisType::enableApplyButton);
     connect(m_ui->mailNotifPassword, &QLineEdit::textChanged, this, &ThisType::enableApplyButton);
+    connect(m_ui->sendTestEmail, &QPushButton::clicked, this, [this]
+    {
+        app()->sendTestEmail();
+        QMessageBox::information(this, tr("Test email"), tr("Email has been sent"));
```

Comment:
>Email has been sent

Misleading to show this even when it fails.

Reply:
I agree with you. I wanted to provide some confirmation to the user, since the interaction feels indeterminant without it. But the smtp code runs async, so we don't yet know if the message will send successfully. Maybe the message should be altered?

Reply:
> Maybe the message should be altered?

Perhaps you could squeeze an `attempted` into the notification? 

```
Email message has been attempted```

Reply:
>Maybe the message should be altered?

"An attempt to send an email has been made"? 
I don't have a good suggestion...

Reply:
I went with "Attempted to send email. Check your inbox to confirm success".